cypress-io/cypress
# fix: allow running tests outside Vite project root folder

- Reference https://github.com/cypress-io/cypress/issues/22505 (I'm not sure it solves all the problems in that issue)

### Additional details
See explanation here: https://github.com/cypress-io/cypress/issues/22505#issuecomment-1426111689

Note that I'm not sure where `@fs` comes from, probably a Vite plugin?

### Steps to test
_You'll need Yarn 1, unluckily we're still stuck with that_

- clone https://github.com/IlCallo/quasar and checkout `upgrade-cypress-ae`
- `cd ui`
- `yarn install`
- `cd dev`
- `yarn install`
- `cd ..`
- `yarn test:component`
- notice Cypress can correctly detect tests
- click on any test, I did my tests on `QAvatar` ones
- notice the request returns 404, due to a wrong relative path composition

### How has the user experience changed?

We're now able to reference tests outside Vite project root folder in Quasar monorepo

### PR Tasks
